Misc/NEWS.d/next/Library/2022-04-26-18-02-44.gh-issue-91928.V0YveU.rst

```diff
@@ -0,0 +1 @@
+Add `datetime.UTC` alias for `datetime.timezone.utc`.
```

Comment:
Do you want to add "Patch by Kabir Kwatra" here? Up to you, but many people do it.